src/pages/HomePage/HomePage.tsx

```diff
@@ -18,6 +20,10 @@ export const HomePage: React.FC = () => {
   const [tabletsArr, setTabletsArr] = useState<Product[]>([]);
   const [accessoriesArr, setAccessoriesArr] = useState<Product[]>([]);
 
+  const allProducts = [...phonesArr, ...tabletsArr, ...accessoriesArr];
+  const bestPriceProducts = getBestPrice(allProducts);
```

Comment:
looks like `getBestPrice`  should be with `s` at the end (`getBestPrices`), or something like `sortByPrice`